linux_os/guide/system/software/sudo/sudo_require_reauthentication/rule.yml

```
@@ -2,8 +2,7 @@ documentation_complete: true
 
 prodtype: ol7,ol8,ol9,rhel7,rhel8,rhel9,sle12,sle15,ubuntu2204
 
-title: 'The operating system must require Re-Authentication when using the sudo command.
- Ensure sudo timestamp_timeout is appropriate - sudo timestamp_timeout'
+title: 'Require re-authentication when using the sudo command'
```

```suggestion
title: 'Require Re-Authentication When Using the sudo Command'
```
This must be in title case, per the [style guide](https://complianceascode.readthedocs.io/en/latest/manual/developer/04_style_guide.html#rule-sections)